rosdep/python.yaml

```diff
@@ -5225,7 +5225,7 @@ python-websocket:
   debian: [python-websocket]
   fedora: [python-websocket-client]
   gentoo: [dev-python/websocket-client]
-  openembedded: [python3-websockets@meta-python]
+  openembedded: [python3-websocket-client@meta-python]
```

Comment:
This one should actually be:
'${PYTHON_PN}-websocket-client@meta-python'

Reply:
Will do. This is only for `python-websocket`? `python3-websocket` is OK?

Reply:
yes only here (and the apostrophes are important)

PYTHON_PN is either python- or python3- based on selected python version, so when both versions are available then we use this variable (python3-websockets was an exception because there is no recipe for python2 version), but now there are both for websocket-client.